mapwfs.c

```diff
@@ -3376,6 +3377,7 @@ int msWFSGetFeature(mapObj *map, wfsParamsObj *paramsObj, cgiRequestObj *req,
   papszGMLGroups = (char**) calloc(sizeof(char*), map->numlayers);
   papszGMLIncludeItems = (char**) calloc(sizeof(char*), map->numlayers);
   papszGMLGeometries = (char**) calloc(sizeof(char*), map->numlayers);
+  papszWFSDefaultMatchCase = (char**)calloc(sizeof(char*), map->numlayers);
```

Comment:
I don't see this memory being freed anywhere. This should be free in erroneous situations as well.